_specifications/lsp/3.18/language/inlineCompletion.md

```diff
@@ -135,16 +135,16 @@ export namespace InlineCompletionTriggerKind {
 	 * Completion was triggered explicitly by a user gesture.
 	 * Return multiple completion items to enable cycling through them.
 	 */
-	export const Invoke: 0 = 0;
+	export const Invoke: 1 = 2;
```

Comment:
Does this compile?

Reply:
Good catch. It is a spec document so no compiler. Fixed it.